src/Lykke.AlgoStore.CSharp.AlgoTemplate.Services/Functions/ADX/AdxFunction.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Lykke.AlgoStore.CSharp.Algo.Core.Candles;
+using Lykke.AlgoStore.CSharp.Algo.Core.Functions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Lykke.AlgoStore.CSharp.AlgoTemplate.Services.Functions.ADX
+{
+    public class AdxFunction : IFunction
+    {
+        private readonly int _period;
+        private int _samples;
+        private Candle _previousInput;
+        private AdxParameters _functionParams = new AdxParameters();
```

Comment:
_previousInput field is assigned in the code but is not used. Is it needed ? 
I believe "new AdxParameters();" is ignored during initialization and is not needed here.

Reply:
This is standard structure and I should use it in this way